functions.sh

```diff
@@ -32,6 +32,9 @@ function get_arch() {
     s390x)
       arch="s390x"
       ;;
+    arm64)
+      arch="amd64"
```

Comment:
why the mismatch arch here?

Reply:
I'm also confused here.

Reply:
dunno, but it produces the correct diff

Reply:
aaah, this should be arm64 just like the one below

Reply:
see OP - that gives the wrong diff